tests/Feature/Generators/ModelGeneratorTest.php

```diff
@@ -481,8 +481,8 @@ public function output_generates_models_with_custom_namespace_correctly()
         $path = 'app/Models/Tag.php';
         $model = 'models/custom-models-namespace.php';
 
-        $this->app['config']->set('blueprint.models_namespace', 'Models');
-
+        // $this->app['config']->set('blueprint.models_namespace', '');
+        // TODO: Review this?
```

Comment:
I commented out the config for setting namespace because the custom namespace should 'just work' given it matches the new default namespace.